doc/Programming-guide/Using-Kaa-endpoint-SDKs/index.md

```diff
@@ -11,9 +11,7 @@ In order to save developers' time, allow users concentrate on important business
 An endpoint SDK is a library that provides communication, data marshaling, persistence, and other functions available in Kaa for a specific [SDK type]({{root_url}}Glossary/#sdk-type).
 Each SDK type is designed to be embedded into your [client application]({{root_url}}Glossary/#kaa-client) and works in conjunction with [Kaa cluster]({{root_url}}Glossary/#kaa-cluster) that serves as a cloud-based middleware for a particular IoT solution.
 
-
-
-The table below captures the list of officially supported platforms and the recommended Kaa SDKs.
+The following table provides platforms that were verified to support at least one type of the Kaa SDK (C, C++, Java, or Objective C). Click in an appropriate cell for the detailed instructions for your target platform and programming language.
```

Comment:
Remove list of SDK types from the first phrase: "(C, C++, Java, or Objective C)".